Analytics Pull Request

stephentmoon opened the pull request.

stephentmoon committed:
Update DatasetAuthorCount.rq

stephentmoon committed:
Update DatasetCount.rq

stephentmoon committed:
Update DatasetCountryCount.rq

stephentmoon committed:
Update MemberCount.rq

stephentmoon committed:
Update OrganizationWithMemberCount.rq

stephentmoon committed:
Update PublicationAuthorCount.rq

stephentmoon committed:
Update PublicationCount.rq

stephentmoon committed:
Update PublicationCountryCount.rq

stephentmoon committed:
Update xml-to-html-statistics.xslt

mrpatrickwest commented on queries/PublicationCount.rq:
```diff
@@ -1,9 +1,8 @@
 #Counts the total number of publications on the DCO website
-#
+#Stephen Moon, July 2015
 SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
```
Do we need a distinct in front of ?publication

stephentmoon replied:
Just tested it out, both give the same result (1532 publications).

On Mon, Jul 20, 2015 at 11:16 AM, Patrick West notifications@github.com
wrote:

> In queries/PublicationCount.rq
> https://github.com/tetherless-world/dco-twsparql/pull/2#discussion_r35005599
> :
> 
> >  #Counts the total number of publications on the DCO website
> > -#
> > +#Stephen Moon, July 2015
> >  SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
> 
> Do we need a distinct in front of ?publication
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tetherless-world/dco-twsparql/pull/2/files#r35005599.

## 

Stephen T. Moon
(860) 480-8751

mrpatrickwest replied:
Go ahead and add distinct. Might not matter now, but could later. Just to be on the safe side.

mrpatrickwest commented on xslt/xml-to-html-statistics.xslt:
```diff
@@ -18,7 +18,7 @@ Version 1 : Dave Beckett (DAWG)
 Customization for SPARQler: Andy Seaborne
 Customization for TWC: Patrick West
-Modified by Stephen Moon for DCO
+Modified by Stephen Moon for DCO Statistics
```
Is the header template even being called? If not then let's remove it.

stephentmoon committed:
Update xml-to-html-statistics.xslt

stephentmoon committed:
Update xml-to-html-statistics.xslt

stephentmoon committed:
Create DCOMembersByCommunity.rq

Sorts DCO members by their communities

mrpatrickwest commented on xslt/xml-to-html-statistics.xslt:
```diff
@@ -119,7 +110,6 @@ Modified by Stephen Moon for DCO
 
   <xsl:template match="res:sparql">
 	<xsl:if test="res:head/res:link">
```
Remove the empty if statement

mrpatrickwest commented on xslt/xml-to-html-statistics.xslt:
```diff
@@ -32,15 +32,6 @@ Modified by Stephen Moon for DCO
-      </xsl:for-each>
-    </div>
-  </xsl:template>
-
```
below here, line 58 or so, there's a td tag that needs to be removed along with its ending tag. We're not using tables in this xslt

mrpatrickwest replied:
also below about line 46 in the vb-result template there's a div tag that needs to be removed and its ending tag. We don't need that either. We can always add it to the page if we'd like to.

stephentmoon committed:
Update xml-to-html-statistics.xslt

stephentmoon committed:
Update xml-to-html-statistics.xslt

Commit:
Updated the members query to sort by community name, removed datatype at the
end of the number.

Added comments to -cont, -date, -desc to distinguish them from the original
xml-to-html.xsl.

Renamed xml-to-html-statistics.xsl removing the t at the end

Commit:
Added distinct for ?publication

Commit:
resolving conflict qith PublicationCount.rq
